www/pages/case-studies/hulu.amp.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import Head from 'next/head';
+import { useAmp } from 'next/amp';
+
+import Page from '../../components/page';
+import Footer from '../../components/footer';
+import SocialMeta from '../../components/social-meta';
+import Hero from '../../components/case-studies/hulu/hero';
+import Banner from '../../components/case-studies/hulu/banner';
+import Intro from '../../components/case-studies/hulu/intro';
+import Gallery from '../../components/case-studies/hulu/gallery';
+import Testimony from '../../components/case-studies/hulu/testimony';
+import Improvements from '../../components/case-studies/hulu/improvements';
+import LinesGallery from '../../components/case-studies/hulu/lines-gallery';
+import Conclusion from '../../components/case-studies/hulu/conclusion';
+import CTA from '../../components/case-studies/hulu/cta';
+
+const title = 'Case Study - Hulu | Next.js';
+
+export default () => {
+  const isAmp = useAmp();
+
+  return (
+    <Page title={title}>
+      {isAmp && (
+        <Head>
+          <script
+            async
+            custom-element="amp-carousel"
+            src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"
+          />
+        </Head>
+      )}
+      <SocialMeta
+        title={title}
+        description="Find out how Hulu migrated to Nextjs and modernized their fronted stack to ship code faster."
```

Comment:
Is `Nextjs` intentional?

```suggestion
        description="Find out how Hulu migrated to Next.js and modernized their fronted stack to ship code faster."
```